src/main/java/uk/gov/hmcts/reform/et/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -73,10 +76,49 @@ public static Boolean isHearingIsLinkedFlag(CaseData caseData) {
         return Boolean.parseBoolean(caseData.getHearingIsLinkedFlag());
     }
 
-    public static String getTribunalAndOfficeLocation(CaseData caseData) {
-        return caseData.getTribunalAndOfficeLocation();
+    public static String getTribunalAndOfficeLocation() {
+        return null;
     }
 
+    public static HearingWindow getHearingWindow() {
+        HearingWindow hearingWindow = new HearingWindow();
+        hearingWindow.setDateRangeStart(null);
```

Comment:
Necessary to set to null? Can you use builder instead?

Reply:
Refactored